src/lib/flow/codegen/blocklayer/prelude.ts

```
@@ -0,0 +1,145 @@
+			let registerFunc: Node[] = [
+				{
+					type: ReprEnum.LocalFunctionDeclaration,
+					funcdecl: {
+						type: ReprEnum.FunctionDeclaration,
+						name: "register",
+						params: [],
+						body: [registerFnNode],
+						returnType: {}
+					}
+				}
+			];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)